### DISCUSSION TITLE
The EUP5108 Microcontroller

### DISCUSSION THREAD

**Discussion by Emil-DV** [logisim-evolution/logisim-evolution#1169] [Show and tell]
I've been a software/systems engineer for years and I always wanted to design a micro-controller along the lines of the Intel 8051.  When I first came upon Logisim-Evolution a few months ago I finally got the chance to re-imagine what a minimalist (small transistor count) 8 bit CISC processor would be if I started from scratch.

The result is the EUP5108 8-bit Virtual Micro-controller and a series of YouTube videos about it.
https://www.youtube.com/playlist?list=PLutzSUqCeqd2JNwKN7Za1qZU8AJ8HDwoR

**Comment by maehne** [comment]
Thanks for the link! It's a nice series of videos showcasing Logisim-evolution's capabilities.

**Comment by Emil-DV** [comment]
We have an Assembler!
I finally got around to writing an Assembler for the EUP5108 so my modified version of logisim v3.5 is no longer needed to run the initial .ROM files.  I've also tested and confirmed that it all works in the latest (3.7.2) version.  For more information check out my Pateron page: https://www.patreon.com/eunumpluribus
